cdap-app-fabric/src/test/java/co/cask/cdap/config/UserPreferencesStoreTest.java

```diff
@@ -230,14 +230,14 @@ public void testMultiLevelProperties() throws Exception {
     store.deleteProperties("myspace", "app");
     Assert.assertTrue(store.getProperties("myspace", "app").isEmpty());
     propMap.put("key", "program");
-    store.setProperties("myspace", "app", "type", "prog", propMap);
-    Assert.assertEquals(propMap, store.getProperties("myspace", "app", "type", "prog"));
+    store.setProperties("myspace", "app", "workflow", "prog", propMap);
```

Comment:
how come these changes are required?

Reply:
these are added because previously I added some tests here, but I think we should use correct program type for the test